src/uct/api/uct.h

```diff
@@ -261,23 +261,38 @@ struct uct_iface_attr {
         struct {
             size_t           max_short;  /**< Maximal size for put_short */
             size_t           max_bcopy;  /**< Maximal size for put_bcopy */
-            size_t           max_zcopy;  /**< Maximal size for put_zcopy */
-        } put;
+            size_t           max_zcopy;  /**< Maximal size for put_zcopy (total
+                                              of @ref uct_iov_t::length of the
+                                              @a iov parameter) */
+            size_t           max_iov;    /**< Maximal @a size of @ref ::uct_iov_t
+                                              @a iov parameter in @ref ::uct_ep_put_zcopy
+                                              @anchor uct_iface_attr_cap_put_max_iov */
+        } put;                           /**< Attributes for PUT operations */
 
         struct {
             size_t           max_bcopy;  /**< Maximal size for get_bcopy */
-            size_t           max_zcopy;  /**< Maximal size for get_zcopy */
-        } get;
+            size_t           max_zcopy;  /**< Maximal size for get_zcopy (total
+                                              of @ref uct_iov_t::length of the
+                                              @a iov parameter) */
+            size_t           max_iov;    /**< Maximal @a size of @ref ::uct_iov_t
```

Comment:
@yosefe I would propose to get only one max_iov variable  instead max_iov for each type of the operation  AM,  PUT, GET.